polyfill/test/Date/prototype/minus/subclass-invalid-result.js

```diff
@@ -0,0 +1,25 @@
+// Copyright (C) 2020 Igalia, S.L. All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+
+/*---
+esid: sec-temporal.date.minus
```

Comment:
Needs Symbol.species feature for consistency with the other tests where we specify it? (We discussed on a previous PR that it was optional, but it should be consistent.) Seems to apply to a few other tests as well.